frontend/components/MainPage/sections/AlertSections/UpdateAvailableAlert.tsx

```diff
@@ -47,7 +49,7 @@ export const UpdateAvailableAlert = () => {
 
       return comparison === SemverComparisonResult.OUTDATED;
     },
-    refetchInterval: 1000 * 60 * 5, // 5 minutes
+    refetchInterval: ONE_MINUTE_INTERVAL * 5, // 5 minutes
```

Comment:
would rather do 
```suggestion
    refetchInterval: FIVE_MINUTES_INTERVAL
```

or at least can remove comment, but up to you